### PR TITLE
Doc updates related to changes to vic-machine create options

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic/issues/1284, https://github.com/vmware/vic/issues/1309, https://github.com/vmware/vic/issues/877, https://github.com/vmware/vic/issues/1008, https://github.com/vmware/vic/issues/871. 

Updated the descriptions of the vic-machine create options, the deployment instructions, examples.

### CODE REVIEW

**Comment by andrewtchin** [comment]
lgtm

File: doc/user_doc/vic_installation/vch_installer_examples.md [hunk 1]
```diff
@@ -21,16 +21,15 @@ You can install vSphere Integrated Containers directly on an ESXi host that is n
...
 --user root
---compute-resource /ha-datacenter/host/localhost.eng.vmware.com/Resources/
 --image-datastore <i>datastore_name</i>
 </pre>
 
 <a name="cluster"></a>
 ## Deploy a Virtual Container Host in a vCenter Server Cluster ##
 
-If vCenter Server manages clusters, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
+If vCenter Server manages more than one cluster, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
```
Comment (emlin) [inline doc/user_doc/vic_installation/vch_installer_examples.md]: Actually, here we did enhancement to make it easier.
So relative path is accepted, for example, you can specify path as host/Resources/path/somewhere.
Or in ESXi, you could just write the path after Resources, for example path/somewhere
In VC, that could be 'vc cluster'/path/somewhere

File: doc/user_doc/vic_installation/vch_installer_options.md [hunk 2]
```diff
@@ -150,17 +213,26 @@ Use this option in combination with the `cert` option, that provides the path to
...
-**NOTE**: If your password contains special characters, you must wrap the password in single quotation marks (').
+**NOTE**: If your password contains special characters, you must wrap the password in single quotation marks (') on Mac OS and Linux and in double quotation (") marks on Windows.
 
 <pre>--password '<i>esxi_host_or_vcenter_server_p@ssword</i>'</pre>
 
 ### `force` ###
+
+Short name: `-f`
+
 Forces `vic-machine` to ignore warnings and non-fatal errors and continue with the deployment of a virtual container host. Errors such as an incorrect compute resource still cause the installation to fail. Using the `force` option deletes any existing virtual container host appliances or datastore folders that have the same name as the one you are specifying in the current deployment.
```
Comment (emlin) [inline doc/user_doc/vic_installation/vch_installer_options.md]: The delete function of force install is removed, cause we already had vic-machine delete, that one is more clear and not easy to cause unexpected result.

Comment (stuclem) [inline doc/user_doc/vic_installation/vch_installer_options.md]: Deleted the sentence about deleting an existing installation.

**Comment by stuclem** [comment]
[skip ci]

**Comment by stuclem** [comment]
@emlin I have updated the following topics to correct the paths for compute-resource:

https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/vch_installer_options.md
https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/vch_installer_examples.md

I also updated the confirmation message after a successful VCH deployment in https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/install_vic_cli.md.

Is this OK now?
Thanks!

**Comment by stuclem** [comment]
:recycle: just in case...

**Comment by mdubya66** [comment]
lgtm

**Comment by stuclem** [comment]
:recycle:

**Pull request merged**